src/main/java/view/UI.java

```diff
@@ -0,0 +1,13 @@
+package view;
+
+/**
+ * UI is an abstract class providing a common structure for user interface classes.
+ */
+public abstract class UI {
```

Comment:
Is it UI or Ui? 
![image](https://github.com/nus-cs2103-AY2324S2/ip/assets/111892633/4dfdc497-a4fb-49d3-95d8-6bd27a22aea5)

```suggestion
public abstract class Ui {
```